docs/history.md

```diff
@@ -1,3 +1,7 @@
+## 2.24.0
+* Fix getBlockAt when outside bounds
+* Improve documentation and examples
+
```

Comment:
can you add : 
* Add ability to change the skin parts of a bot (thanks @Naomi-alt )

Reply:
sure :) done